hw/formal/data/fpv.mk

```diff
@@ -0,0 +1,44 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+.DEFAULT_GOAL := all
+
+all: build
+
+###################
+## build targets ##
+###################
+build: compile_result
+
+gen_sv_flist:
+	@echo "[make]: gen_sv_flist"
+	cd ${build_dir} && ${sv_flist_gen_cmd} ${sv_flist_gen_opts}
+
+pre_compile: gen_sv_flist
+	@echo "[make]: pre_compile"
+	mkdir -p ${build_dir} && env | sort > ${build_dir}/env_vars
+	-cp -Ru ${tool_srcs} ${sv_flist_gen_dir}
+
+compile: pre_compile
+	@echo "[make]: compile"
+	# we check the status in the parse script below
+	cd ${sv_flist_gen_dir} && ${build_cmd} ${build_opts} | tee ../fpv.log
+
+post_compile: compile
+	@echo "[make]: post_compile"
+
+# Parse out result
+compile_result: post_compile
+	@echo "[make]: compile_result"
+	${report_cmd} ${report_opts}
+
+clean:
+	echo "[make]: clean"
+	rm -rf ${scratch_root}/${dut}/*
```

Comment:
How does this path relate to `$(build_dir)`? It seems odd that the `clean` target deletes something that isn't explicitly created anywhere else in the Makefile. Also, I would have thought it's better to delete the directory, rather than all non-hidden files inside it (rarely the right thing to do).

Reply:
I see, thanks Rupert. That is a good point. Usually "build_dir" is within the "scratch_root".
I believe all the ".mk" files have this line. @sriyerg @msfschaffner. 
1). This seems a little bit stale.. We do not use `${scratch_root}/${dut}/` anymore. 
2). Are we using this "clean" function anywhere? Seems like in '-purge' we just use `os.remove` to remove the directory.
Do you think we can delete this one?

Reply:
this is something we've carried over from the initial `sim.mk` file, but I agree that it looks stale. let's wait for @sriyerg to confirm.

Reply:
Agreed - this can be removed.